### DISCUSSION TITLE
Using prompt weights in arrays

### DISCUSSION THREAD

**Discussion by ILikeToasters** [lllyasviel/Fooocus#2432] [General]
Was curious if any way of using prompt weights inside an array or if it is possible to add that in the future? It seems like using parentheses inside the array will not work. It would be a useful thing for testing different prompt weights with the same seed.

**Comment by mashb1t** [comment]
@flannerybh maybe you can support here as the creator of the array feature.

**Reply by ILikeToasters** [reply]
I did some further digging and did notice that there might be a pull request for improving the wildcards that would allow for them to be used in order with the same seed and since it seems that you can use prompt weights in wildcards that might be an option. But would be nice in arrays as well.

**Comment by flannerybh** [comment]
> Was curious if any way of using prompt weights inside an array or if it is possible to add that in the future? It seems like using parentheses inside the array will not work. It would be a useful thing for testing different prompt weights with the same seed.

Yeah, I noticed someone else tried to use parentheses in an array and didn't work. Nether do decimals and other characters. Can you make an issue and ping me in it?

**Reply by mashb1t** [reply]
https://github.com/lllyasviel/Fooocus/issues/2437
Parentheses around the array work perfectly fine.
